src/screen/unix.rs

```diff
@@ -256,13 +254,13 @@ impl Terminal {
     }
 
     fn translate_bytes(bytes: Vec<u8>) -> Vec<Key> {
-        if bytes == [27, '[' as u8, 'A' as u8] { return vec![Up] };
-        if bytes == [27, '[' as u8, 'B' as u8] { return vec![Down] };
-        if bytes == [27, 'O' as u8, 'A' as u8] { return vec![Up] };
-        if bytes == [27, 'O' as u8, 'B' as u8] { return vec![Down] };
+        if bytes == [27, b'[' as u8, b'A' as u8] { return vec![Up] };
+        if bytes == [27, b'[' as u8, b'B' as u8] { return vec![Down] };
+        if bytes == [27, b'O' as u8, b'A' as u8] { return vec![Up] };
+        if bytes == [27, b'O' as u8, b'B' as u8] { return vec![Down] };
```

Comment:
I left it because there can be architectures where character literals are actually `i8` instead of `u8`. Haven't met them yet in the wild, though.